src/v/config/configuration.h

```diff
@@ -468,7 +468,7 @@ struct configuration final : public config_store {
     property<std::chrono::milliseconds>
       cloud_storage_topic_purge_grace_period_ms;
     property<bool> cloud_storage_disable_upload_consistency_checks;
-    property<bool> cloud_storage_disable_metadata_consistency_checks;
+    deprecated_property cloud_storage_disable_metadata_consistency_checks;
```

Comment:
I'm confused about this change. Isn't this configuration option what is used to, among other things, allow uploads to proceed when certain conditions arise, such as a gap in the log due to retention expiring data in a TS disabled state? If so, how is it related to fencing?

Since the option is a boolean, and apparently the value doesn't matter any more, should there also be dead code removed?

Reply:
> No, this is a different config option. 

haha ok, thanks. lots of escape hatches!